conf/locale/uk/LC_MESSAGES/django.po

```diff
@@ -25776,3 +25786,25 @@ msgstr "http://desk.ua/etichniy-kodeks"
 msgid "http://desk.ua/en/contact"
 msgstr "http://desk.ua/contact"
 
+#: lms/djangoapps/certificates/models.py
+msgid "already received"
+msgstr "вже отримано"
+
+msgid "didn't receive"
+msgstr "не отримано"
+
+msgid "error states"
+msgstr ""
+
+msgid "audit passing states"
+msgstr ""
+
+msgid "audit not passing states"
+msgstr ""
+
+#: common/lib/xmodule/xmodule/graders.py
+msgid "{short_label} Avg"
+msgstr "{short_label} Сер."
+
+msgid "{section_type} Average = {percent:.0%}"
```

Comment:
https://www.eudusa.org/index.php?title=%D0%A1%D0%BF%D0%B8%D1%81%D0%BE%D0%BA_%D0%9E%D1%81%D0%BD%D0%BE%D0%B2%D0%BD%D0%B8%D1%85_%D0%A1%D0%BA%D0%BE%D1%80%D0%BE%D1%87%D0%B5%D0%BD%D1%8C